packages/translate/lib/__tests__/__snapshots__/testLineResult.test.tsx.snap

```diff
@@ -879,3 +935,18 @@ Object {
   "value": "App seems to have exited correctly, however another app has been opened",
 }
 `;
+
+exports[`Test line results translation translate "openAppOverrideFailed" render with addition errorType 1`] = `
+"✖ Open application
+  fail: Open app override failed"
+`;
+
+exports[`Test line results translation translate "openAppOverrideFailed" render with addition errorType 2`] = `
+"✖ Open application
+  fail: Open app override failed"
+`;
+
+exports[`Test line results translation translate "openAppOverrideFailed" render with lineId 1`] = `
+"✖ Open application
+  fail: Open app override failed on line line-id-1"
```

Comment:
Please, make this more generic, no need to display line id. This info would not make any difference for the user anyway